Northstar.Client/mod/resource/fontfiletable.txt

```diff
@@ -0,0 +1,40 @@
+FontFileTable
+{
+	"Default"					"resource/MetronicPro-Regular.vfont"	[!$JAPANESE && !$TCHINESE]
+	"Default"					"resource/NotoSansJP-Regular.vfont"		[$JAPANESE]
+	"Default"					"resource/NotoSansTC-Regular.vfont"		[$TCHINESE]
+
+	"DefaultBold"				"resource/MetronicPro-SemiBold.vfont"	[!$JAPANESE && !$TCHINESE]
+	"DefaultBold"				"resource/NotoSansJP-Regular.vfont"		[$JAPANESE]
+	"DefaultBold"				"resource/NotoSansTC-Regular.vfont"		[$TCHINESE]
+
+	"Titanfall"					"resource/Titanfall-Regular.vfont"		[!$JAPANESE && !$TCHINESE && !$RUSSIAN]
+	"Titanfall"					"resource/NotoSansJP-Regular.vfont"		[$JAPANESE]
+	"Titanfall"					"resource/NotoSansTC-Regular.vfont"		[$TCHINESE]
+	"Titanfall"					"resource/MetronicPro-SemiBold.vfont"	[$RUSSIAN]
+
+	"marlett"					"vgui/fonts/marlett.ttf"
+
+	// Everything below is DONOTSHIP / Dev only
```

Comment:
Uh, is this still relevant?

Reply:
Turns out that comment was added by Respawn in the original file inside the VPK :,)